README.md

```diff
@@ -52,17 +52,26 @@ End to end example of how to use `s3torchconnector` can be found under the
 
 The simplest way to use the S3 Connector for PyTorch is to construct a dataset, either a map-style or iterable-style 
 dataset, by specifying an S3 URI (a bucket and optional prefix) and the region the bucket is located in:
-```shell
+```py
```

Comment:
The IDE suggests `python` when trying to update this section.

Reply:
I have tried to turn on rich view, it works with just `py`